File: tl/src/main/java/com/github/badoualy/telegram/tl/api/messages/TLAllStickersNotModified.java
package com.github.badoualy.telegram.tl.api.messages;

/**
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
public class TLAllStickersNotModified extends TLAbsAllStickers {
    public static final int CONSTRUCTOR_ID = 0xe86602c3;

    public TLAllStickersNotModified() {
    }

    @Override
    public String toString() {
        return "messages.allStickersNotModified#e86602c3";
    }

    @Override
    public int getConstructorId() {
        return CONSTRUCTOR_ID;
    }

    @Override
    @SuppressWarnings("PointlessBooleanExpression")
    public boolean equals(Object object) {
        if (!(object instanceof TLAllStickersNotModified)) return false;
        if (object == this) return true;

        TLAllStickersNotModified o = (TLAllStickersNotModified) object;

        return true;
    }
}

File: tl/src/main/kotlin/com/github/badoualy/telegram/tl/api/messages/TLAllStickersNotModified.kt
package com.github.badoualy.telegram.tl.api.messages

import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_BOOLEAN
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_CONSTRUCTOR_ID
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_DOUBLE
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_INT32
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_INT64
import com.github.badoualy.telegram.tl.TLObjectUtils.computeTLBytesSerializedSize
import com.github.badoualy.telegram.tl.TLObjectUtils.computeTLStringSerializedSize
import kotlin.Any
import kotlin.Boolean
import kotlin.Int
import kotlin.String

/**
 * messages.allStickersNotModified#e86602c3
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
class TLAllStickersNotModified : TLAbsAllStickers() {
    private val _constructor: String = "messages.allStickersNotModified#e86602c3"

    override val constructorId: Int = CONSTRUCTOR_ID

    override fun toString() = _constructor

    override fun equals(other: Any?): Boolean {
        if (other !is TLAllStickersNotModified) return false
        if (other === this) return true

        return true
    }
    companion object  {
        const val CONSTRUCTOR_ID: Int = 0xe86602c3.toInt()
    }
}
